### DISCUSSION TITLE
Nimrod Mk1

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#48] [Card Balance]
EQUIPMENT FIREARM
S +4

**Comment by Sol-au-fein** [comment]
Nimrod Autocannon is the heaviest of heavy weapons. It is capable of burst fire.

I like original card

Suggestion: 
EQUIPMENT FIREARM
S +4
1 Action: Gain another S +4 until end of your turn. Stackable.

**Reply by 21-ric** [reply]
It's similar to "SR Mk. XII 'Assailant'" but more powerfull. 

SR Mk. XII 'Assailant' (IMPERIAL) [R]
EQUIPMENT, FIREARM
[S] +4. 1 Action: +2 to [S] (until end of [S] COMBAT)

In my opinion "assailant" is balanced. What do you think? 🙂
The goal is to have these cards playable.

**Reply by Sol-au-fein** [reply]
Yeah, i didnt know about Assailant. So original nimrod is way stronger. That cannot be.

Or maybe there will be some revamp of sniper rifles in future and assailant will be different

**Comment by Kite80** [comment]
Nimrod should be the most powerful weapon in human arsenal, right after gehenna puker. It's a big nasty autocannon with an impressive fire rating; the concept was "the more you fire at your target, the bigger the damage to it" and the paper version represented it well. Maybe in digital DT it could get a different skill, like the ability to shoot an area and hit one different target for each action spent for the attack.

**Reply by Sol-au-fein** [reply]
I think that is exactly ability of Capitol Nimrod MK2. I feel +6 on Gehenna Puker is too high.

**Comment by 21-ric** [comment]
EQUIPMENT FIREARM
[S] +4. After a WARRIOR is WOUNDED or KILLED with the NIMROD MK I, this card gain +1 to [S].

**Comment by Michalx11** [comment]
I do agree Nimrod is THE big gun. Based on lore it is currently THE BEST HMG on the market. Thst is one gun that I think should have SLAYER. However in order to not making it too OP in propose:

EQUIPMENT FIREARM
+4 S
Attach for 2 Actions.
Spend 2 Actions for Attack and gain SLAYER in Shoot until end of Combat

**Comment by david-roxygen** [comment]
Nice the last version

EQUIPMENT FIREARM
+4 S
Attach for 2 Actions.
Spend 2 Actions for Attack and gain SLAYER in Shoot until end of Combat
